images/src/ContentReactor.Images/ContentReactor.Images.Services/ImagesService.cs

```diff
@@ -52,30 +56,32 @@ public ImagesService(IBlobRepository blobRepository, IImageValidatorService imag
             var imageId = Guid.NewGuid().ToString();
 
             // create a blob placeholder (which will not have any contents yet)
-            var blob = BlobRepository.CreatePlaceholderBlob(FullImagesBlobContainerName, userId, imageId);
+            var blob = BlobRepository.CreatePlaceholderBlob(FullImagesBlobContainerName, imageId);
 
             // get a SAS token to allow the client to write the blob
-            var writePolicy = new SharedAccessBlobPolicy
+            BlobSasBuilder blobSasBuilder = new BlobSasBuilder
             {
-                SharedAccessStartTime = DateTime.UtcNow.AddMinutes(-5), // to allow for clock skew
-                SharedAccessExpiryTime = DateTime.UtcNow.AddHours(24),
-                Permissions = SharedAccessBlobPermissions.Create | SharedAccessBlobPermissions.Write
+                StartsOn = DateTime.UtcNow.AddMinutes(-5),
```

Comment:
The clock skew comment seems helpful. We may want to keep that here to explain why we're rolling back by 5 minutes.

Reply:
Resolved